tests/UtestTest.cpp

```diff
@@ -232,6 +232,33 @@ TEST(UtestShell, DestructorIsCalledForLocalObjectsWhenTheTestFails)
 
 #endif
 
+#ifndef CPPUTEST_USE_LONG_LONG
+// otherwise these are tested by UnitTestMacros
```

Comment:
True, but that doesn't reduce the NLOC for the tests, since this tests the disabled case while the other tests the enabled case, and they behave differently.